lib/configuration.js

```diff
@@ -166,6 +193,13 @@ function applyConfiguration(inConfig) {
     },
   };
 
+  // Transform apiDescriptions from strings to an array of objects
+  outConfig.apiDescriptions = coerceToArray(inConfig.apiDescriptions)
+    .map((apiDescription, i) => ({
+      location: `configuration.apiDescriptions[${i}]`,
+      content: apiDescription,
+    }));
```

Comment:
I'd like to have more separation between what user provides to Dredd and what's Dredd's resolved, internal configuration, but that requires to pretty much revamp how Dredd gets configured. So doing this for now, coercing user's input to a data structure Dredd uses internally.